bin/micro

```diff
@@ -79,7 +79,12 @@ if ('function' !== typeof mod) {
   process.exit(1)
 }
 
-const { port, host } = args
+let { port, host } = args
+
+if (typeof port === 'boolean') {
```

Comment:
maybe some docs here would be nice, to explain that the minimist behavior when no args are specified is to set it to `true`. Otherwise it's kinda non-obvious what's going on here